clients/drcachesim/scheduler/scheduler_impl.cpp

```diff
@@ -154,7 +154,7 @@ scheduler_impl_tmpl_t<memref_t, reader_t>::get_reader(const std::string &path,
         }
         for (; iter != end; ++iter) {
             const std::string fname = *iter;
-            if (fname == "." || fname == ".." ||
+            if (fname == "." || fname == ".." || fname == DRMEMTRACE_AUX_DIRNAME ||
```

Comment:
directory_iterator_t says "// Iterates over files: skips sub-directories.".  So you should remove the check for the AUX_DIRNAME as it will just confuse future readers wondering whether it skips sub-dirs or not.